gitea-mirror - install broken, repo moved, needs update

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

Gitea-Mirror

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/gitea-mirror.sh)"

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 📝 Provide a clear and concise description of the issue.

Script fails and exits error code 1.

after a quick peek, it appears the github repo arunavo4/gitea-mirror moved to RayLabsHQ/gitea-mirror like 4 hours ago. So the script needs to be updated for:

replace:
https://api.github.com/repos/arunavo4/gitea-mirror/releases/latest

with:
https://api.github.com/repos/RayLabsHQ/gitea-mirror/releases/latest

and whatever other instances of the original repo url are in the script obv should also be replaced similarly. I'm not set up to do that right now (and honestly don't have the bandwidth at the moment) or I would just do it myself

### 🔄 Steps to reproduce the issue.

Step 1: run Gitea-Mirror helper script on your proxmox host shell
Step 2: receive error code 1 because the script is looking for files in a repo that no longer hosts them 😅

### ❌ Paste the full error output (if available).

  ✔️   Installed Dependencies
######################################################################## 100.0%                   
bun was installed successfully to /opt/bun/bin/bun 

Added "/opt/bun/bin" to $PATH in "~/.bashrc" 

To get started, run: 

  source /root/.bashrc 
  bun --help 
  ✔️   Installed Bun
curl: (22) The requested URL returned error: 404
curl: (22) The requested URL returned error: 404
curl: (22) The requested URL returned error: 404
   ✖️   Failed to fetch release metadata from https://api.github.com/repos/arunavo4/gitea-mirror/releases/latest after 3 attempts

[ERROR] in line 31: exit code 0: while executing command return 1

[ERROR] in line 1249: exit code 0: while executing command lxc-attach -n "$CTID" -- bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/install/"$var_install".sh)" $?

### 🖼️ Additional context (optional).

_No response_

--- a/ct/gitea-mirror.sh
+++ b/ct/gitea-mirror.sh
@@ -3,7 +3,7 @@ source <(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxV
 # Copyright (c) 2021-2025 community-scripts ORG
 # Author: CrazyWolf13
 # License: MIT | https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
-# Source: https://github.com/arunavo4/gitea-mirror
+# Source: https://github.com/RayLabsHQ/gitea-mirror
 
 APP="gitea-mirror"
 var_tags="${var_tags:-mirror;gitea}"
@@ -28,7 +28,7 @@ function update_script() {
     msg_error "No ${APP} Installation Found!"
     exit
   fi
-  RELEASE=$(curl -fsSL https://api.github.com/repos/arunavo4/gitea-mirror/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
+  RELEASE=$(curl -fsSL https://api.github.com/repos/RayLabsHQ/gitea-mirror/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
   if [[ "${RELEASE}" != "$(cat ~/.${APP} 2>/dev/null || cat /opt/${APP}_version.txt 2>/dev/null)" ]]; then
 
     msg_info "Stopping Services"
@@ -48,7 +48,7 @@ function update_script() {
     msg_ok "Installed Bun"
 
     rm -rf /opt/gitea-mirror
-    fetch_and_deploy_gh_release "gitea-mirror" "arunavo4/gitea-mirror"
+    fetch_and_deploy_gh_release "gitea-mirror" "RayLabsHQ/gitea-mirror"
 
     msg_info "Updating and rebuilding ${APP} to v${RELEASE}"
     cd /opt/gitea-mirror

--- a/frontend/public/json/gitea-mirror.json
+++ b/frontend/public/json/gitea-mirror.json
@@ -9,9 +9,9 @@
   "updateable": true,
   "privileged": false,
   "interface_port": 4321,
-  "documentation": "https://github.com/arunavo4/gitea-mirror/",
+  "documentation": "https://github.com/RayLabsHQ/gitea-mirror/",
   "config_path": "/etc/systemd/system/gitea-mirror.service",
-  "website": "https://github.com/arunavo4/gitea-mirror/",
+  "website": "https://github.com/RayLabsHQ/gitea-mirror/",
   "logo": "https://cdn.jsdelivr.net/gh/selfhst/icons/webp/gitea-mirror.webp",
   "description": "Gitea Mirror auto-syncs GitHub repos to your self-hosted Gitea, with a sleek Web UI and easy Docker deployment. ",
   "install_methods": [

--- a/install/gitea-mirror-install.sh
+++ b/install/gitea-mirror-install.sh
@@ -3,7 +3,7 @@
 # Copyright (c) 2021-2025 community-scripts ORG
 # Author: CrazyWolf13
 # License: MIT | https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
-# Source: https://github.com/arunavo4/gitea-mirror
+# Source: https://github.com/RayLabsHQ/gitea-mirror
 
 source /dev/stdin <<<"$FUNCTIONS_FILE_PATH"
 color
@@ -28,7 +28,7 @@ ln -sf /opt/bun/bin/bun /usr/local/bin/bun
 ln -sf /opt/bun/bin/bun /usr/local/bin/bunx
 msg_ok "Installed Bun"
 
-fetch_and_deploy_gh_release "gitea-mirror" "arunavo4/gitea-mirror"
+fetch_and_deploy_gh_release "gitea-mirror" "RayLabsHQ/gitea-mirror"
 
 msg_info "Installing gitea-mirror"
 cd /opt/gitea-mirror

